### ISSUE TITLE
Fix explain_array.rst_ links in docs

### ISSUE THREAD

**Issue by meghanrjones** [GenericMappingTools/gmt#4624]
**Description of the problem**

Currently, the documentation pages for ```filter1d```, ```gmtmath```, ```gmtregress```, ```grdinterpolate```, ```histogram```, ```pshistogram```, ```makecpt```, and ```sample1d``` include doc/rst/source/explain_array.rst_, which demonstrates the use of options for creating 1-D arrays via ```gmtmath```.

Here is an example of the [generate 1D array section in the makecpt docs](https://docs.generic-mapping-tools.org/6.1/makecpt.html#generate-1d-array).

I think the 1D array section only makes sense in the ```gmtmath``` docs and would like to submit a PR with that section removed from the other modules' documentation.

**Comment by PaulWessel** [comment]
I agree it uses gmtmath to illustrate how to set up a series of numbers.  But we do need something like this in filter1d, sample1d, etc, etc. to let people know how to to use -T.  Perhaps the file could need some language to say we are illustrating the effect of -T using gmt math but that the same syntax is used throughout GMT wherever a 1-D series need to be indicated?  Alternatively, introduce a much shorter explain*.rst_ file for those other modules but have it include a link to the full one for more details?

**Comment by meghanrjones** [comment]
Many of the other options that require additional explanation include references to sections of the cookbook, **-r[g|p]** is a good example. Is that an option? Otherwise, revising to make it more clear that the purpose is to explain **-T** would also work.

**Comment by PaulWessel** [comment]
Yes, that is a possibility.  We would have to basically place most of that doc file into the general features sections for how to create 1-d series, then refer to it.
